File: AkitaModel/ApiError.cs
namespace Skyware.Lis.AkitaModel;

/// <summary>
/// API error. Usually sent as content in case of non-success HTTP codes, such as 500, 404, etc.
/// </summary>
public class ApiError
{

    /// <summary>
    /// Correlation Identifier.
    /// </summary>
    public string CorrelationId { get; set; }

    /// <summary>
    /// Reason of the error.
    /// </summary>
    public string Message { get; set; }

    /// <summary>
    /// Detailed information, e.g. exception stack.
    /// </summary>
    public string Details { get; set; }

}

File: AkitaMotelTS/src/ApiError.ts
/**
 * Skyware.Lis.AkitaModel
 */

/**
 * API error. Usually sent as content in case of non-success HTTP codes, such as 500, 404, etc.
 */
export class ApiError {
  /**
   * Correlation Identifier.
   */
  correlationId?: string;

  /**
   * Reason of the error.
   */
  message?: string;

  /**
   * Detailed information, e.g. exception stack.
   */
  details?: string;
}
